README.MD

```diff
@@ -92,6 +92,8 @@ mail_server:
 
 For Humboldt organization you must ask for the master keys file which you must add to ```config/credentials```, it is inside the internal repository.
```

Comment:
Ajustar "...you must copy the encrypted files and the master keys from the internar repository to `config/credentials`."